### DISCUSSION TITLE
How to increase webhook rate limit ?

### DISCUSSION THREAD

**Discussion by cherrycharan** [Budibase/budibase#7401] [Help]
Current there are only 10 request that webhook can process per sec and where can we increase the number . We are using budibase on k8s

**Comment by cherrycharan** [comment]
@shogunpurple Can you help me on this pls ?

Sorry for tagging directly and asking

**Reply by burkeazbill** [reply]
Was there ever any response or action taken on this? I'm running the free budibase via docker-compose. If I only send 1 or 2 webhooks at a time to it, they work flawlessly.. unfortunately, my actual use case was to receive events from the VMware Lab Platform which can send 10-50 events within a second ( I receive e-mails for every single one without issue ), but very few actually get processed by my budibase instance. Yesterday, I tried 4 and only 3 were received. Today I tried 16 and only 4 were received. With this very low level of success, I must revert to e-mail based integration :(
